README.md

```diff
@@ -4,6 +4,34 @@
 ## Source Control Infrastructure
```

Comment:
I would move this to a ```git_tutorial_readme.MD``` as this file, ```README.md``` would get confused with the preexisting README.MD